### ISSUE TITLE
Light / Dark mode button is not accessible 

### ISSUE THREAD

**Issue by rakelbjort** [doczjs/docz#1359]
## Bug Report

**Describe the bug**
The button in the upper right corner for switching between light or dark mode for the application is not accessible for screen readers

**Expected behavior**

Please consider adding an aria-label on the button.

Suggestion "Switch to dark mode" / "Switch to light mode"
